package.json

```diff
@@ -26,13 +26,12 @@
   "main": "lib/redux.js",
   "unpkg": "dist/redux.js",
   "module": "es/redux.js",
-  "typings": "./index.d.ts",
+  "typings": "./es/index.d.ts",
```

Comment:
This should still be output in the root of the package. The types are always the same.

Reply:
OK, I'll see if I can make it do this and push a new commit

Reply:
OK, I changed it to output type declarations into the `types/` directory and to use them for everything.

With this change, this PR will need to be merged before #3524